src/main/java/ru/l1ttleO/boyfriend/Utils.java

```diff
@@ -16,81 +28,195 @@ public static String getBeep() {
         return "Б%sп!".formatted(randomElement(letters));
     }
 
-    public static <T> T plural(final long amount, final T first, final T second, final T five) {
+    public static <T> T plural(long amount, final T one, final T two_to_four, final T five_to_zero) {
```

Comment:
```suggestion
    public static <T> T plural(long amount, final T one, final T twoToFour, final T fiveToZero) {
```
`Parameter name 'two_to_four' doesn't match regex '[a-z][A-Za-z\d]*'`
`Parameter name 'five_to_zero' doesn't match regex '[a-z][A-Za-z\d]*'`